docs/source/parsing_best_practices.md

```diff
@@ -0,0 +1,240 @@
+# nvstrings Parsing Best Practices
+
+
+## Regex Performence Tips:
```

Comment:
Performance is misspelled twice that I can find.

Reply:
Did a spell check again and fixed this.  Hopefully, there are no more typo's .